Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/common/connectionmanager/persistence/HibernateUtil.java

```diff
@@ -49,7 +50,8 @@
     static {
         try {
             // Create the SessionFactory from hibernate.cfg.xml
-            sessionFactory = new Configuration().configure(getConfigFile()).buildSessionFactory();
+            sessionFactory = new Configuration().configure()
+                    .buildSessionFactory(new StandardServiceRegistryBuilder().configure(getConfigFile()).build());
         } catch (HibernateException he) {
             // Make sure you log the exception, as it might be swallowed
             System.err.println("Initial SessionFactory creation failed." + he);
```

Comment:
Optional: Please remove or replace any codes that use "System.err" or "System.out"